static/app/views/performance/transactionSummary/transactionEvents/content.tsx

```diff
@@ -0,0 +1,242 @@
+import * as React from 'react';
+import {Fragment} from 'react';
+import {browserHistory} from 'react-router';
+import styled from '@emotion/styled';
+import {Location, Query} from 'history';
+import omit from 'lodash/omit';
+
+import {CreateAlertFromViewButton} from 'app/components/createAlertButton';
+import TransactionsTable from 'app/components/discover/transactionsTable';
+import SearchBar from 'app/components/events/searchBar';
+import GlobalSdkUpdateAlert from 'app/components/globalSdkUpdateAlert';
+import * as Layout from 'app/components/layouts/thirds';
+import {getParams} from 'app/components/organizations/globalSelectionHeader/getParams';
+import Pagination from 'app/components/pagination';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import DiscoverQuery from 'app/utils/discover/discoverQuery';
+import EventView from 'app/utils/discover/eventView';
+import {decodeScalar} from 'app/utils/queryString';
+import {stringifyQueryObject, tokenizeSearch} from 'app/utils/tokenizeSearch';
+import {Actions, updateQuery} from 'app/views/eventsV2/table/cellAction';
+import {TableColumn} from 'app/views/eventsV2/table/types';
+
+import {getCurrentLandingDisplay, LandingDisplayField} from '../../landing/utils';
+import {SpanOperationBreakdownFilter} from '../filter';
+import TransactionHeader, {Tab} from '../header';
+
+import {generateTraceLink, generateTransactionLink} from './utils';
+
+const DEFAULT_TRANSACTION_LIMIT = 12;
+
+type Props = {
+  location: Location;
+  eventView: EventView;
+  transactionName: string;
+  organization: Organization;
+  isLoading: boolean;
+  totalValues: Record<string, number> | null;
+  projects: Project[];
+  spanOperationBreakdownFilter: SpanOperationBreakdownFilter;
```

Comment:
Is this used for anything?

Reply:
Hey Tony, I've deleted `isLoading` and `totalValues` as they are completely unused. `projects` is still required for some header functionality so I've kept that. I've removed `spanOperationBreakdownFilter` as I'm not currently using it but I suspect it may need to return in some form when I start attaching filter logic when linking to the Events tab from other tables.

Reply:
Yeah, let's add it back once these props are being used.